index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css">
+    <title>Awesome Books -JavaScript Project</title>
+</head>
+<body>
+    <header>
+        <nav class="nav-bar">
+            <div>Awesome Books</div>
+            <div class="nav-list">
+               <div class="nav-items list-nav"><a href="#">List</a></div>
+               <div class="nav-items add-nav"><a href="#">Add new</a></div>
+               <div class="nav-items contact-nav"><a href="#">Contact</a></div>
+            </div>
+        </nav>
+       <div class="date"><p id="date-para"></p></div> 
+    </header>
+    <main>
+    <section>
+    <h1>Welcome to My bookstore</h1>
```

Comment:
- [ ] 👍 Well! Upon reviewing your project, I noticed that the "Welcome to My Bookstore" h1 title is showing in all sections of the webpage. However, if you look closely at the design, this title should only be visible in the "Books list" section. Please make sure to follow the design carefully and update your code accordingly by updating all those section title. 📚🏷️

![image](https://github.com/Armynerh/awesome_books-es6/assets/55840999/c7dc3100-d406-44fa-8f44-9c48a6c2fd46)